project_close_forecast_date/wizard/project_close_month.py

```diff
@@ -0,0 +1,16 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields
+
+
+class ProjectCloseMonth(models.TransientModel):
+    _name="project.close.month.wizard"
+    _description="project.close.month.wizard"
+
+    project_ids = fields.Many2many("project.project")
+    last_close_date = fields.Date()
```

Comment:
@ChristianSantamaria  a last close date could be unset, or should this field be required?